rest-api/src/main/java/io/quarkus/qe/services/RepositoryService.java

```diff
@@ -31,6 +35,11 @@ public Repository findById(Long id) throws RepositoryNotFoundException {
         return repositoryMarshaller.fromEntity(entity);
     }
 
+    public List<Repository> findAll(final int from, final int to) {
+        PanacheQuery<RepositoryEntity> query = RepositoryEntity.findAll();
+        return repositoryMarshaller.fromEntities(query.range(from, to).list());
```

Comment:
We're building the list twice and we could map the entities directly by using:

```
return RepositoryEntity.findAll().page(0, 20).stream().map(RepositoryMarshaller::fromEntity).list();
```

Moreover, I think it's more readable this way.